### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 19

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#345] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325400-332ed77b-4554-42e8-bd03-1479cde45be7.png)

**Comment by i0nut02** [comment]
<img width="772" alt="Schermata 2022-02-05 alle 14 33 28" src="https://user-images.githubusercontent.com/99051485/152644270-4cd2fcd1-8fc9-4500-9ea0-f8c1456b31b6.png">

**Comment by i0nut02** [comment]
riguardo l'esercizio, non so se è sat o unsat (ad occhio direi unsat) siccome non me lo chiede non lo controllo neanche,

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217818305-652394eb-28bd-4d8b-8ca9-98365305ea1d.png)

**Comment by francesco-alberghina** [comment]
![immagine](https://github.com/sapienzastudentsnetwork/mmi2122/assets/68157574/2fb0a3d3-2ab5-4a13-8b4a-48948bbcf013)
